examples/pi.c

```diff
@@ -98,12 +96,22 @@ int main(int argc, char *argv[]) {
   }
   end = clock();
 
-  mp_int_to_string(&sum1, g_radix, g_buf, sizeof(g_buf));
+  size_t g_buf_size = ndigits + 2; /* "3" prefix and terminating '\0' */
```

Comment:
```suggestion
  size_t buf_size = ndigits + 2; /* "3" prefix and terminating '\0' */
```

Since this is no longer global, let's please drop the `g_` prefix.